File: src/chapter06/oo/MathUtil.java
package chapter06.oo;

public class MathUtil {

	private MathUtil() {
		super();
	}
	
	public static int min(int a, int b, int c) {
	
		if (a < b && a < c) {
			return a;
		}
		
		if (b < c) {
			return b;
		}
		
		return c;
	}
	
	public static int max(int a, int b, int c) {
		
		if (a > b && a > c) {
			return a;
		}
		
		if (b > c) {
			return b;
		}
		
		return c;
	}
	
}

File: lab/04-classes/mathutil.py

class MathUtil:

    @staticmethod
    def min(a, b, c):
        if (a < b and a < c):
            return a
        elif b < c:
            return b
        return c

    @staticmethod
    def max(a, b, c):
        if (a > b and a > c):
            return a
        elif b > c:
            return b
        return c

if __name__ == "__main__":
    print ("Maximum von 1,2,3:", MathUtil.max(1,2,3))
    print ("Minimum von 1,2,3:", MathUtil.min(1,2,3))
